.wp-env/mu-plugin.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+// Mimic w.org for testing wporg-two-factor
```

Comment:
It'd be good to remove all this from the `README` now that it's automated, and add the steps to setup this new env.